[KOGITO-8159] Event state not starting state if exclusive=false (#243)
kiegroup/kogito-docs: serverlessworkflow/modules/ROOT/pages/getting-started/cncf-serverless-workflow-specification-support.adoc

The link:{spec_doc_url}#parallel-state[Parallel State] of the workflow states feature works in a single thread. This means that a Parallel State does not create one thread per branch, simulating an actual parallel behavior.

> The link:{spec_doc_url}#event-state[Event State] of the workflow states feature should not be used as starting state if exclusive property is set to `false`. In case is specified that way it will behave as if exclusive was set to `true`. 

Reviewer: ```suggestion
 If an exclusive property is set to `false`, you should not use the link:{spec_doc_url}#event-state[Event State] of the workflow states feature as the starting state. In case, if it is specified that way, then it will behave as if an exclusive property was set to `true`. 
```
